src/formatDate.js

```diff
@@ -50,7 +50,55 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const FIRST_SPLITTER = fromFormat[3];
+
+  const dateSplit = date.split(FIRST_SPLITTER);
+
+  let year = '';
+
+  if (fromFormat.indexOf('YYYY') !== -1) {
```

Comment:
```suggestion
  if (fromFormat.indexOf(LONG_YEAR_FORMAT) !== -1) {
```
Consider using variables with meaningful names instead of using magic strings